JM/CF/Scripts/1_Core/CommunityFramework/Files/CF_Directory.c

```diff
@@ -0,0 +1,106 @@
+class CF_Directory
+{
+	/**
+	 * @param pattern The search pattern
+	 * @param [inout] files An array where the list of files will be written to
+	 * @param flags If it should be looked in directories or .pak files, or both
+	 * 
+	 * @return If the pattern was valid
```

Comment:
Tested and no the return value has nothing to say about if the pattern itself was valid. FindFile returns a NULL handle if the pattern was valid but there was no file that matched it. So instead this line should be changed to:

`* @return If any files were found that matched the pattern`